monero-wallet-gui.pro

```diff
@@ -317,7 +317,7 @@ TRANSLATIONS =  \ # English is default language, no explicit translation file
                 $$PWD/translations/monero-core_he.ts \ # Hebrew
                 $$PWD/translations/monero-core_ko.ts \ # Korean
                 $$PWD/translations/monero-core_ro.ts \ # Romanian
-                $$PWD/translations/monero-core_da.ts \ # Danish
```

Comment:
Same of above, doing this the Danish translation will result invalidated, please fix